### PR TITLE
#16 TjSynchronized, TjSynchronizedTest

### PR DESCRIPTION
**Pull request by l3r8yJ**
@yegor256 have a look, please
Here are the tests that do not pass TjDefault and TjSync implementation

### CODE REVIEW

File: src/main/java/com/yegor256/tojos/TjSynchronized.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    @Override
+    public Tojo add(final String name) {
+        synchronized (this.wrapped) {
+            return this.wrapped.add(name);
+        }
+    }
+
+    @Override
+    public List<Tojo> select(final Predicate<Tojo> filter) {
+        synchronized (this.wrapped) {
```
Comment (yegor256) [inline src/main/java/com/yegor256/tojos/TjSynchronized.java:62]: @l3r8yJ I think it's redundant. There is no reason to synchronize the reading.

**Comment by yegor256** [comment]
@l3r8yJ see my comment above

**Comment by l3r8yJ** [comment]
@yegor256 fixed

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**